src/registry/cosignV1/edit.tsx

```diff
@@ -0,0 +1,129 @@
+
+/**
+ * Form to configure a Formio 'coSign' (cosign v1) type component.
+ */
+const EditForm: EditFormDefinition<CosignV1ComponentSchema> = () => {
+  const intl = useIntl();
+  useDeriveComponentKey();
+  const {errors} = useFormikContext<CosignV1ComponentSchema>();
+
+  const erroredFields = Object.keys(errors).length
+    ? getErrorNames<CosignV1ComponentSchema>(errors)
+    : [];
+  // TODO: pattern match instead of just string inclusion?
+  // TODO: move into more generically usuable utility when we implement other component
+  // types
+  const hasAnyError = (...fieldNames: string[]): boolean => {
+    if (!erroredFields.length) return false;
+    return fieldNames.some(name => erroredFields.includes(name));
+  };
+
+  return (
+    <Tabs>
+      <TabList>
+        <BuilderTabs.Basic
+          hasErrors={hasAnyError('label', 'description', 'authPlugin', 'hidden')}
+        />
+        <BuilderTabs.Translations hasErrors={hasAnyError('openForms.translations')} />
+      </TabList>
+
+      {/* Basic tab */}
+      <TabPanel>
+        <Label />
+        <Description />
+        <AuthPlugin />
+        <Hidden />
+      </TabPanel>
+
+      {/* Translations */}
+      <TabPanel>
+        <Translations.ComponentTranslations<CosignV1ComponentSchema>
+          propertyLabels={{
+            label: intl.formatMessage(LABELS.label),
+            description: intl.formatMessage(LABELS.description),
+          }}
+        />
+      </TabPanel>
+    </Tabs>
+  );
+};
+
+/*
+  Making this introspected or declarative doesn't seem advisable, as React is calling
+  React.Children and related API's legacy API - this may get removed in future
+  versions.
+
+  Explicitly specifying the schema and default values is therefore probbaly best, at
+  the cost of some repetition.
+ */
+EditForm.defaultValues = {
+  // basic tab
+  label: '',
+  key: '',
+  description: '',
+  hidden: false,
+  authPlugin: '',
+};
+
+export interface AuthPluginOption {
+  id: string;
+  label: string;
+}
+
+function isAuthPluginOptions(
+  options: AuthPluginOption[] | undefined
+): options is AuthPluginOption[] {
+  return options !== undefined;
+}
+
+export const AuthPlugin: React.FC = () => {
+  const {getAuthPlugins} = useContext(BuilderContext);
+  const {value: options, loading, error} = useAsync(async () => await getAuthPlugins(), []);
+  if (error) {
+    throw error;
+  }
+  const _options = isAuthPluginOptions(options) ? options : [];
+  return (
+    <Select
+      name="authPlugin"
+      label={
+        <FormattedMessage
+          description="Label for 'authPlugin' builder field"
+          defaultMessage="Authentication method"
+        />
+      }
+      description={
+        <FormattedMessage
+          description="Description for 'authPlugin' builder field"
+          defaultMessage={`Which authentication method the co-signer must use. Note that
+          this must be an authentication method available on the form.`}
```

Comment:
Why not use a <kbd>?</kbd> instead? Is it to have the note displayed to the user by default?

Reply:
Correct. It contains an important constraint that should always be visible, as it affects what is valid and what is not, whereas a tooltip is usually informational.